sql/core/src/main/scala/org/apache/spark/sql/execution/SparkStrategies.scala

```diff
@@ -310,4 +311,17 @@ private[sql] abstract class SparkStrategies extends QueryPlanner[SparkPlan] {
       case _ => Nil
     }
   }
+
+  object DDLStrategy extends Strategy {
```

Comment:
@yhuai,i write a draft version for this, can you have a look(https://github.com/scwf/spark/compare/apache:master...scwf:createDataSourceTable?expand=1)

why we put ``` case r: RunnableCommand => ExecutedCommand(r)``` in BasicOperators is because we no need make a new strategy for only one rule. 

And after we refactor command implementation in spark sql, we should make the newly added command follow ```RunnableCommand``` if possible then we can avoid adding new strategy for newly added command.

/cc @marmbrus